### PR TITLE
✨ Added basic dropdown with utilities

### PR DESCRIPTION
**Pull request by thaoms**
## To-do's

- [x] Add wrapper that holds functionality (default dropdown)

### CODE REVIEW

File: src/dropdowns/DropdownItem.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement } from 'react';
+
+type DropdownItem = {
```
Comment (cedricengagor) [inline src/dropdowns/DropdownItem.tsx:3]: PP: Would probably rename to Props or something. This way it wont collide with the actual component when we'd have to export it 🙇

File: src/dropdowns/DropdownItem.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    onClick={props.callback ?? undefined}
+                    className={props.selected
+                        ? listItemActiveClasses.join(' ')
+                        : listItemRegularClasses.join(' ')}
+                >
+                    <span className={`flex pl-${props.level * 24}`}>
+                        {props.children}
+                    </span>
+                </div>
+            }
```
Comment (cedricengagor) [inline src/dropdowns/DropdownItem.tsx:61]: Q: Would it make sense to apply the padding on the `option` and pass the correct html element in the usage of the dropdown? This way the dropdown doesn't handle much except for the visual aspect 🙇

**Review by cedricengagor — COMMENTED** [review]
Lets also think about the way that we would use this component. At the moment we don't have anything that renders it based on the trigger. Maybe we should add a component there for this as well. Or maybe we should still interpret this in the dropdown. Cause I think if we give too much freedom for this people will misuse the component 🙇

**Review by cedricengagor — APPROVED** [review]

**Review by matthias-deschoenmacker — APPROVED** [review]
Code looks good to me

File: src/Dropdowns/DropdownItem.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        'text-charcoal-color-400',
+        'hover:bg-primary-color-200',
+        'dark:text-white',
+        'dark:hover:text-charcoal-color-400',
+    ];
+
+    const listItemActiveClasses = [
+        ...listItemBaseClasses,
+        'text-white',
+        'bg-primary-color-600',
```
Comment (marcusrelacion) [inline src/Dropdowns/DropdownItem.tsx:35]: `bg-primary-color-600` should be `bg-primary-color-500`

File: src/Dropdowns/DropdownList.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+const DropdownList = forwardRef<HTMLDivElement, PropsWithChildren<unknown>>(
+    function DropdownList(
+        { children },
+        ref
+    ) {
+        return (
+            <div
+                aria-live="polite"
+                aria-atomic="true"
+                className="shadow rounded-8 bg-white dark:bg-charcoal-color-500 min-w-320 max-w-400 overflow-hidden"
```
Comment (marcusrelacion) [inline src/Dropdowns/DropdownList.tsx:13]: `rounded-8` should be `rounded-4`

File: src/Dropdowns/DropdownItem.tsx (hunk 3 above)
Comment (marcusrelacion) [reply src/Dropdowns/DropdownItem.tsx:35]: Not seeing where the border color is set for the divider, but the divider should use `border-charcoal-300`.

**Review by marcusrelacion — CHANGES_REQUESTED** [review]

**Review by goOve — APPROVED** [review]

**Pull request merged**